modules/ROOT/pages/configure-tls-context-rtf-lb.adoc

```diff
@@ -5,9 +5,52 @@ endif::[]
 
 All inbound traffic entering Anypoint Runtime Fabric is encrypted using TLS. When enabling inbound traffic, you must provide a valid TLS certificate. When you configure inbound traffic for Runtime Fabric, you can configure the TLS server by importing a Privacy-Enhanced Mail (PEM) or Java Keystore (JKS) file, or by importing a TLS context from the secrets manager (advanced use case).
```

Comment:
>All inbound traffic entering Anypoint Runtime Fabric is encrypted using TLS.

This is no longer true with the Multiple Virtual Servers UI.

Reply:
Updated - thank you!